### ISSUE TITLE
UDT member not valid in Debug.Print

### ISSUE THREAD

**Issue by KarlSevenSeven** [WaynePhillipsEA/twinbasic#300]
```
Module MainModule
    
    Type TB_BlocksA
        X As Double
        Y As Double
    End Type
    
   Dim Blocks As TB_BlocksA
    
    ' This project type is set to 'Standard EXE' in the Settings file, so you need a Main() subroutine to run when the EXE is started.
    Public Sub Main()
        
        Dim DD As Boolean
        
        DD = True
        
        With Blocks
        .X = 123
        .Y = 456
        If DD Then Debug.Print "X,Y",.X,.y
        End With
        
    End Sub
	
End Module
```

In the Debug.Print line, the y member throws a compile error.
If , .y is commented out, no error occurs.

Just run it.

**Desktop (please complete the following information):**
 - OS:  Windows 10
 - twinBASIC v0.10.3423

---

BTW, how do we find out the TwinBasic version quickly?

**Comment by WaynePhillipsEA** [comment]
Confirmed, thanks for reporting.  For the version number you can get it from the debugger Callstack panel:

<img width="282" alt="tb_callstackVersionNumber" src="https://user-images.githubusercontent.com/25738109/127325337-1f6a3543-7506-4f05-b513-22e724205a57.png">

Or from the Extension activity bar:

<img width="248" alt="tb_extensionVersionNumber" src="https://user-images.githubusercontent.com/25738109/127325357-80652f2e-9023-4388-b86c-732e45afb768.png">

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.3613, thanks for reporting!
